helm_deploy/hmpps-book-a-video-link-api/values.yaml

```diff
@@ -22,7 +22,6 @@ generic-service:
     APPLICATIONINSIGHTS_CONNECTION_STRING: "InstrumentationKey=$(APPINSIGHTS_INSTRUMENTATIONKEY)"
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.json
     DB_SSL_MODE: "verify-full"
-    # Add the gov-notify template IDs here when we know what they are.
```

Comment:
It is actually simpler to add the template IDs to the application.yml file.  The end result is the same i.e. they end up as env vars but we can test locally without have to copy them around locally.